sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -38,19 +38,19 @@ private[ui] class AllExecutionsPage(parent: SQLTab) extends WebUIPage("") with L
       if (listener.getRunningExecutions.nonEmpty) {
         _content ++=
           new RunningExecutionTable(
-            parent, "Running Queries", currentTime,
+            parent, "Running Queries (" + listener.getRunningExecutions.size + ")", currentTime,
```

Comment:
Yes, I thought `s"Running Queries (${listener.getRunningExecutions.size})"` one is slightly better.